pkg/globalnet/controllers/service_export_controller.go

```diff
@@ -0,0 +1,155 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package controllers
+
+import (
+	"context"
+
+	"github.com/submariner-io/admiral/pkg/federate"
+	"github.com/submariner-io/admiral/pkg/syncer"
+	"github.com/submariner-io/admiral/pkg/util"
+	submarinerv1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog"
+	mcsv1a1 "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+)
+
+func NewServiceExportController(config syncer.ResourceSyncerConfig) (Interface, error) {
+	var err error
+
+	klog.Info("Creating ServiceExport controller")
+
+	_, gvr, err := util.ToUnstructuredResource(&corev1.Service{}, config.RestMapper)
```

Comment:
```suggestion
	_, gvr, err := util.ToUnstructuredResource(&mcsv1a1.ServiceExport, config.RestMapper)
```
Should it be mcsv1a1.ServiceExport?

Reply:
No, this is used to retrieve `Service` objects.